.rubocop.yml

```diff
@@ -7,10 +7,15 @@ AllCops:
       - 'db/seeds.rb'
       - 'bin/**/*'
       - 'config/initializers/simple_form.rb'
+      - 'spec/rails_helper.rb'
+      - 'lib/vacancy_scraper/north_east_schools/*.rb'
 
 Rails:
     Enabled: true
 
+Rails/HasManyOrHasOneDependent:
+    Enabled: false
+
```

Comment:
Added a card for this to be addressed separately as we need to ensure we understand how we to approach this and if there are any dependencies that we do want to be deleting.

https://trello.com/c/uR5v7Pb6/357-address-rubocop-errors-for-rails-hasmanyorhasonedependent